src/facet.js

```diff
@@ -38,9 +38,10 @@ class Facet extends Mark {
     }
     for (let i = 0; i < this.marks.length; ++i) {
       const mark = this.marks[i];
-      const markFacets = mark.facet == null ? mark.data === this.data ? facetsIndex : undefined
-        : mark.facet === "exclude" ? facetsIndex.map(facet => Uint32Array.from(difference(index, facet)))
-        : mark.facet === true ? facetsIndex
+      const facet = mark.facet && keyword(mark.facet, "facet", ["include", "exclude"]);
```

Comment:
This needs to happen in the mark constructor (as I said previously), not here in initialize.

Reply:
yes I understood your comment just after pushing the previous commit 8-)
I think it's worth adding an "auto" option value (rather than undefined?), which makes the code easier to read, and would maybe clarify the documentation

Reply:
Yea, I wasn’t sure whether we needed `"auto"` in addition to `undefined`, but sure, if you want.